Packs/Core/TestPlaybooks/playbook-Local_Analysis_alert_Investigation_-_Test.yml

```diff
@@ -0,0 +1,934 @@
+id: Local Analysis alert Investigation - Test
+version: -1
+name: Local Analysis alert Investigation - Test
+description: |-
+  This playbook tests the 'Local Analysis alert Investigation' playbook, which is part of the 'Core' pack. The playbook includes the following conducted tests:
+  - Checks playbook run correctly without inputs.
+  - Checks the handling of malicious verdicts.
+  - Verify the execution of containment actions.
+  - Validate the blocking of indicators.
+  - Test raising severity of the incident
+  - Test file detonation
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: a8d7741f-828b-480e-881d-9fca26419369
+    type: start
+    task:
+      id: a8d7741f-828b-480e-881d-9fca26419369
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "22"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -980
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 54c10b5e-0a7f-421e-84ce-9fe9af76c3c5
+    type: playbook
+    task:
+      id: 54c10b5e-0a7f-421e-84ce-9fe9af76c3c5
+      version: -1
+      name: Local Analysis alert Investigation
```

Comment:
Replace it with the same playbook for updating the changes with the input Query.